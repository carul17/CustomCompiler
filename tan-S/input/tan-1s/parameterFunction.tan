subr float f(int a, float b){
	return a + b;
	
}

main {
	const x :=  f(1, 2.0);
	print x\n;
	
	
}