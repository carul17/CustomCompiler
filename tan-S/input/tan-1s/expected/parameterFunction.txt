Bad operand type for LoadI instruction at location 119.
